docs/docs/guides/solution-scenarios/domain-discovery.mdx

```diff
@@ -31,7 +31,7 @@ When opening `login.mycompany.com` then the login policy of the instance will be
 This means that you have to configure the [Login and Access](/docs/guides/manage/console/default-settings#login-behavior-and-access) Policy and [Identity Providers](/docs/guides/manage/console/default-settings#identity-providers) for the **CIAM** users on the instance itself.
 
 :::info
-You can also configure these settings on the default organization (see below) and send the scope `urn:zitadel:iam:org:id:{id}` with every [auth request](/docs/apis/openidoauth/authrequest#organization-policies-and-branding).
+You can also configure these settings on the default organization (see below) and send the scope `urn:zitadel:iam:org:id:{id}` with every [auth request](/docs/oidc-playground#organization-policies-and-branding).
```

Comment:
Are you sure you want to change that one?

Reply:
ok removed 👍